crowbar_framework/app/assets/stylesheets/content/panel.scss

```diff
@@ -0,0 +1,55 @@
+.panel-heading {
+  border-bottom: 0;
+}
+
+.panel .panel-heading h3 {
+  padding: 0;
+  margin: 0;
+}
+.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
+  border: 1px;
+  border-left: 7px;
```

Comment:
Properties should be ordered border, border-color, border-left, border-radius, border-style, box-shadow, margin